totp/__init__.py

```diff
@@ -41,20 +42,27 @@ def get_pass_entry(path):
 
 
 def copy_to_clipboard(text):
-    selection = os.environ.get('PASSWORD_STORE_X_SELECTION', 'clipboard')
     try:
+        if platform.system() == 'Linux':
+            selection = os.environ.get('PASSWORD_STORE_X_SELECTION', 'clipboard')
+            command = ['xclip', '-selection', selection]
+        elif platform.system() == 'Darwin':
+            command = ['pbcopy']
+        else:
```

Comment:
@hobarrera 
Changed to use `pbcopy` on Darwin, `clip` on Windows, `xclip` on any other platforms.